javascript/src/test/components/ChannelMessages/ChannelLinks.test.js

```diff
@@ -0,0 +1,370 @@
+    };
+});
+describe("ChannelLinks tests", () => {
+
+    test("renders without crashing", () => {
+        useSWR.mockReturnValue({
+            data:  []
+        });
+        render(<ChannelLinks messages={[]} />);
+    });
+
+    test("Displays username", () => {
+        useSWR.mockReturnValue({
+            data: [{
+                id: "U017218J9B3",
+                real_name: "Test Person"
+            }]
+        });
+        const exampleMessage = {
+            "type": "message",
+            "subtype": "channel_join",
+            "ts": "1594143066.000200",
+            "user": "U017218J9B3",
+            "text": "<@U017218J9B3> has joined the channel",
+            "channel": "section-6pm",
+            "user_profile": {
+                "real_name": "Test Person"
+            }
+        }
+        const {getAllByText} = render(<ChannelLinks messages={[exampleMessage]}/>);
+        const nameElement = getAllByText(/Test Person/);
+        expect(nameElement).toHaveLength(2);
+    });
+
+    test("Displays username in message", () => {
+        useSWR.mockReturnValue({
+            data: [{
+                id: "U017218J9B3",
+                real_name: "Test Person"
+            }]
+        });
+        const exampleMessage = {
+            "type": "message",
+            "subtype": "channel_join",
+            "ts": "1594143066.000200",
+            "user": "U017218J9B3",
+            "text": "<@U017218J9B3> has joined the channel",
+            "channel": "section-6pm",
+            "user_profile": {
+                "real_name": "Test Person"
+            }
+        }
+        const {getByText} = render(<ChannelLinks messages={[exampleMessage]}/>);
+        setTimeout(function (){
+            const nameElement = getByText(/Test Person/);
+            expect(nameElement).toBeInTheDocument();
+        }, 500)
+    });
+
+    test("Username not found", () => {
+        useSWR.mockReturnValue({
+            data: []
+        });
+        const exampleMessage = {
+            "type": "message",
+            "subtype": "channel_join",
+            "ts": "1594143066.000200",
+            "user": "U017218J9B3",
+            "text": "<@U017218J9B3> has joined the channel",
+            "channel": "section-6pm",
+            "user_profile": {
+                "real_name": "Test Person"
+            }
+        }
+        const {getByText} = render(<ChannelLinks messages={[exampleMessage]}/>);
+        const nameElement = getByText(/@U017218J9B3/);
+        expect(nameElement).toBeInTheDocument();
+
+    });
+
+    test("Only displays link", () => {
+        useSWR.mockReturnValue({
+            data: []
+        });
+        const exampleMessage = {
+            "type": "message",
+            "subtype": "channel_join",
+            "ts": "1594143066.000200",
+            "user": "U017218J9B3",
+            "text": "Office hours at <https://ucsb.zoom.us/j/89220034995?pwd=VTlHNXJpTVgrSEs5QUtlMDdqMC9wQT09>",
```

Comment:
Can we remove the class Zoom link and replace it with a placeholder value? This shouldn't be published.